CHANGELOG.md

```diff
@@ -4,13 +4,13 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ## Unreleased
 
 ### Added
-- 
+-
 
 ### Changed
-- 
+- **cf-layout** Updated main content spacing
```

Comment:
@contolini Fixed, I guess we can  merge and release? Doesn't seem to be anything in line.